wordcloud.py

```diff
@@ -1,30 +1,28 @@
+"""Create word cloud tools."""
 import operator
-import random
 import string
 
-FILENAME = "initial_text.txt"
 WORDS_WITH_VALUE = ["word", "cloud", "punctuation"]
 
 
-def main():
-    """Produce words suitable for processing into a word cloud using random model selection."""
-    raw_text = read_file(FILENAME)
+def main(source_text, model_selection="occurrence"):
```

Comment:
I was thinking, instead of having one function with a `modal_selection` parameter would it be clearer to have different functions, one for each model? Something to consider. Then it is up to the person using your module to decide which function they will use. 

A benefit of this is you can explain exactly what each of your models do clearly, and even have parameters specific to that model (like a list of which words to increase the value of). It also helps with SRP and makes your code more readable.

Reply:
I can try to do this if you want.  However, with the model selection being a parameter passed to the program, doesn't this enable the user to supply one string and try different models on it?  Otherwise, aren't you having to supply the string each time you access a model?